target/linux/mpc85xx/config-6.6

```diff
@@ -111,9 +120,15 @@ CONFIG_GENERIC_TIME_VSYSCALL=y
 CONFIG_GEN_RTC=y
 # CONFIG_GE_IMP3A is not set
 CONFIG_GIANFAR=y
+<<<<<<< HEAD:target/linux/mpc85xx/config-6.6
```

Comment:
Rebasing error.

Reply:
This was noticed and a new version pushed, thanks for catching it too :)

Reply:
sure. it's still present on the first commit.

Reply:
OK, cleaned up